apps/supernova/package.json

```
@@ -0,0 +1,102 @@
+    "immer": "^10.0.0",
+    "interweave": "^13.0.0",
+    "jest": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
+    "luxon": "^3.0.0",
+    "postcss": "^8.4.21",
+    "postcss-url": "^10.1.3",
+    "prop-types": "^15.8.1",
+    "react-test-renderer": "^18.2.0",
+    "sass": "^1.77.5",
+    "shadow-dom-testing-library": "^1.7.1",
+    "tailwindcss": "^3.3.1",
+    "util": "^0.12.4",
+    "zustand": "4.5.5"
+  },
+  "scripts": {
+    "dev": "NODE_ENV=development node esbuild.config.js --serve --watch",
+    "test": "jest",
+    "build": "NODE_ENV=production node esbuild.config.js",
+    "lint": "eslint"
+  },
+  "appProps": {
+    "theme": {
+      "value": "theme-dark",
+      "type": "optional",
+      "description": "Override the default theme. Possible values are theme-light or theme-dark (default)"
+    },
+    "embedded": {
+      "value": "false",
+      "type": "optional",
+      "description": "Set to true if app is to be embedded in another existing app or page.  If set to true the app won't render a page header/footer and instead render only the content. The default value is false."
+    },
+    "endpoint": {
+      "value": "",
+      "type": "required",
+      "description": "Alertmanager API Endpoint URL"
+    },
+    "filterLabels": {
+      "value": null,
+      "type": "optional",
+      "description": "FilterLabels are the labels shown in the filter dropdown, enabling users to filter alerts based on specific criteria. The 'Status' label serves as a default filter, automatically computed from the alert status attribute and will be not overwritten. The labels must be an array of strings. Example: [\"app\", \"cluster\", \"cluster_type\"]"
+    },
+    "silenceExcludedLabels": {
+      "value": null,
+      "type": "optional",
+      "description": "SilenceExcludedLabels are labels that are initially excluded by default when creating a silence. However, they can be added if necessary when utilizing the advanced options in the silence form. The labels must be an array of strings. Example: [\"pod\", \"pod_name\", \"instance\"]"
+    },
+    "silenceTemplates": {
+      "value": null,
+      "type": "optional",
+      "description": "SilenceTemplates are pre-defined silence templates that can be used to scheduled Maintenance Windows. The format consists of a list of objects including description, editable_labels (array of strings specifying the labels that users can modify), fixed_labels (map containing fixed labels and their corresponding values), status, and title."
+    },
+    "predefinedFilters": {
+      "value": null,
+      "type": "optional",
+      "description": "PredefinedFilters are filters applied through TabNavigation in the UI to differentiate between contexts such as Production, QA, and others. They are loaded by default when the application is loaded. The format is a list of objects including name, display name and matchers (map containing matchers as name and expression pairs)"
```

Need to fix this description
```suggestion
      "description": "PredefinedFilters are groups of filters comprised of a list of filter labels and a regex that is used to filter on the values of that label. They can be thought of as filter categories where the filter logic can be more complex than simple label+values pairs. Predefined filters are defined as an array of objects, where each object is a predefined filter. It contains the display name for the UI as well as a list of filter label + value regex pairs which are AND concatenated when they are being evaluated."
```